### PR TITLE
bump: release LogDNA Agent v2.2.0

### PR DESCRIPTION
**Pull request by darinspivey**
This version constains a bump of @logdna/logger

Semver: patch

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.2.0] - January 19, 2021
+### Changed
+- Bump @logdna/logger which includes an `axios` fix for a SSRF vulerability. This bump
```
Comment (smusali) [inline CHANGELOG.md:11]: @darinspivey, can you append "[#219] (https://github.com/logdna/logdna-agent/pull/219)" as well like we did for others?

File: CHANGELOG.md [hunk 2]
```diff
@@ -259,6 +264,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Lint & Style Validations
 
 [Unreleased]: https://github.com/logdna/logdna-agent/compare/2.1.2...HEAD
```
Comment (smusali) [inline CHANGELOG.md:266]: @darinspivey, need to update this too

**Review by smusali — CHANGES_REQUESTED** [review]
@darinspivey, can u also update the versions inside
- `tools/files/darwin/logdna-agent.rb` and
- `tools/files/win32/logdna-agent.nuspec`
? Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (darinspivey) [reply CHANGELOG.md:11]: I don't follow.  This is 219.  The bump includes the vulnerability patch as well as the passthrough change.

Comment (darinspivey) [reply CHANGELOG.md:11]: fixed

**Review by mdeltito — APPROVED** [review]

**Review by smusali — APPROVED** [review]
lgtm!

**Pull request merged**